client/src/main.scss

```diff
@@ -17,6 +17,11 @@ body {
 	font-size: 5vh;
 }
 
+.monotone{
+	-webkit-filter: grayscale(100%);
```

Comment:
OBT comes with autoprefixer so you don't have to worry about writing browser prefixes in your sass